include/nbl/builtin/hlsl/blit/default_compute_alpha_test.comp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#include <nbl/builtin/hlsl/blit/descriptors.hlsl>
+#include <nbl/builtin/hlsl/blit/alpha_test.hlsl>
+#include <nbl/builtin/hlsl/blit/default_compute_common.comp>
+
+[[vk::binding(nbl::hlsl::blit::InImageBinding::value, nbl::hlsl::blit::BlitDescriptorSet::value)]]
+nbl::hlsl::blit::InTextureType inTexture;
+
+[[vk::binding(nbl::hlsl::blit::StatisticsBinding::value, nbl::hlsl::blit::BlitDescriptorSet::value)]]
+RWStructuredBuffer<nbl::hlsl::blit::AlphaStatistics<nbl::hlsl::blit::AlphaBinCount::value> > statistics;
+
+[[vk::push_constant]]
+nbl::hlsl::blit::parameters_t params;
+
+
+[numthreads(nbl::hlsl::blit::WorkgroupSizeX::value, nbl::hlsl::blit::WorkgroupSizeY::value, nbl::hlsl::blit::WorkgroupSizeZ::value)]
+void main(uint32_t3 dispatchThreadID : SV_DispatchThreadID, uint32_t3 groupID : SV_GroupID)
+{
+
+	nbl::hlsl::blit::alpha_test<nbl::hlsl::blit::BlitDimCount::value>(statistics, inTexture, params, dispatchThreadID, groupID);
+}
+
```

Comment:
we no longer require this header, everything is so simple and overridable now, that the `CComputeBlit` can "synthesise" the entry point and compute shader itself.